extension/config.example.json

```diff
@@ -7,7 +7,7 @@
       "displayName": "Mainnet",
       "logo": "",
       "rpcTarget": "https://mainnet.infura.io/v3/d0f317090d6645b6b494ddc6f1cce5ad",
-      "chainCurrencyName": "Ethereum",
```

Comment:
Updated as we use a mix of Ether and Ethereum. I believe Ether is the correct term.